app/controllers/api/user_books_controller.rb

```diff
@@ -0,0 +1,62 @@
+class Api::UserBooksController < ApplicationController
+  before_action :authenticate_user!
+  before_action :get_user_book, only: [:destroy, :update, :show]
+
+  def index
+    books = params[:finish_date] == nil ? UserBook.to_read(current_user) : UserBook.finished(current_user)
+    if books.empty?
+      render json: {errors: "List is empty"}, status: :unprocessable_entity
+    else
+      options = { include: [:book] }
+      render json: UserBookSerializer.new(books, options).serializable_hash
+    end
+  end
+
+  def show
+    render json: @user_book
+  end
+
+  def create
+    user_book = current_user.user_books.create(user_book_params)
+    if user_book.save
+      render json: {
+        message: 'Successfully created!',
```

Comment:
So this `message` is unnecessary. An `http 200` response already is considered successful.